Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -486,6 +488,14 @@ extension ResourceFileElements {
             }
         )
     }
+
+    private static let defaultSpmResourceFileExtensions = ["xib", "storyboard", "xcdatamodeld", "xcmappingmodel", "xcassets", "lproj"]
+
+    private static func defaultResourcePaths(from path: AbsolutePath) -> [AbsolutePath] {
```

Comment:
I'd add a comment here explaining that for `defaultSpmResourceFileExtensions`, the default behavior is to add them automatically, without user having to specify them explicitly